config/initializers/algoliasearch.rb

```diff
@@ -10,12 +10,14 @@ def initialize
   end
 end
 
-if !Rails.env.production? && ENV.fetch('ALGOLIA_APP_ID') == ALGOLIA_PRODUCTION_APP_ID
+if !Rails.env.production? && ENV['ALGOLIA_APP_ID'] == ALGOLIA_PRODUCTION_APP_ID
   raise AlgoliaEnvironmentError.new
 end
 
-AlgoliaSearch.configuration = {
-  application_id: ENV.fetch('ALGOLIA_APP_ID'),
-  api_key: ENV.fetch('ALGOLIA_WRITE_API_KEY'),
-  pagination_backend: :kaminari
-}
+if ENV['ALGOLIA_APP_ID']
```

Comment:
This line makes me worried that we will deploy an app which won't connect to Algolia. It could pass all the tests (because the tests have the env), and be deployed with no Algolia initialization without throwing an error.

Reply:
This would be caught by the smoke test, but that seems a little too late.

Reply:
Maybe it's an acceptable risk